### ISSUE TITLE
WCAG2Mobile: Draft feedback - 1.3.2 Meaningful Sequence

### ISSUE THREAD

**Issue by ChrisLoiselle** [w3c/wcag2ict#647]
(Mary Jo) 

- May want to comment whether MATF is given the leeway to provide techniques in their document, as the draft note they added is very technique-centric. 

(Chris L) 

1. The note provided by MATF could also apply to 1.3.1 in terms of grouping.
2. Definition of semantic containers should be added to the definition of terms in MATF's glossary.
